src/Calculator/Views/DateCalculator.xaml

```diff
@@ -417,35 +417,23 @@
             </Grid.RowDefinitions>
 
             <!-- From Date -->
-            <TextBlock x:Name="Date_FromLabel"
-                       x:Uid="Date_FromLabel"
-                       Grid.Row="1"
-                       Margin="0,0,0,2"
-                       Foreground="{ThemeResource SystemControlPageTextBaseHighBrush}"
-                       TextWrapping="Wrap"/>
             <CalendarDatePicker x:Name="DateDiff_FromDate"
                                 Grid.Row="2"
                                 Style="{StaticResource DateCalculation_CalendarPickerStyle}"
-                                AutomationProperties.LabeledBy="{x:Bind Date_FromLabel}"
                                 CalendarViewStyle="{StaticResource DateCalculation_CalendarViewStyle}"
                                 Closed="CalendarFlyoutClosed"
-                                DateChanged="FromDate_DateChanged"/>
+                                DateChanged="FromDate_DateChanged"
+                                Header="From"/>
```

Comment:
In order to make sure Calculator works great across all languages, we avoid putting string resources directly in the XAML. Instead, we reference a list of strings in a separate resource file that we translate into all supported languages so that we can pull from the right language based on the user's locale. Notice how we did not specify the `Text` property directly in the TextBlocks we are removing (i.e., "Date_FromLabel")?

So here is what you need to do for each CalendarDatePicker:
- Add `x:Uid` to each CalendarDatePicker so we can localize the header property (e.g., "DateDiff_FromHeader")
- We can reuse the existing resources, since we are using the same strings but just in a different way. In the resource file "/strings/en-us/Resources.resw", update the resource names of the TextBlock's we are removing to the new UID we are adding to CalendarDatePicker. Here is an example.

Before:

```
<data name="Date_FromLabel.Text" xml:space="preserve">
  <value>From</value>
  <comment>From Date label for Date Picker</comment>
</data>
```

After:

```
<data name="DateDiff_FromHeader.Header" xml:space="preserve">
  <value>From</value>
  <comment>From Date header for Date Picker</comment>
</data>
```